ci/terraform/oidc/authdev1.tfvars

```diff
@@ -5,9 +5,9 @@ dns_state_role                              = null
 shared_state_bucket                         = "di-auth-development-tfstate"
 test_clients_enabled                        = "true"
 ipv_api_enabled                             = true
-ipv_authorisation_callback_uri              = ""
-ipv_authorisation_uri                       = ""
-ipv_authorisation_client_id                 = ""
+ipv_authorisation_callback_uri              = "http://localhost:3000/authorize"
```

Comment:
Don't think this should be localhost in what we merge? (Can always change it locally when we need it)

Reply:
Separately, I wonder if it's worth putting in these variables for the authdev envs yet, since I think it only really works in dev and we'll probably need to do some other stuff to get it working in these envs?